plugins/scoreboard/Cargo.toml

```diff
@@ -0,0 +1,9 @@
+[package]
+name = "scoreboard"
+version = "0.1.0"
+edition = "2021"
+
+[dependencies]
+bevy = { workspace = true }
```

Comment:
probably good to explicitly opt into `bevy_ui` here? (though I imagine sickle_ui does for us...?)